.github/workflows/test.yaml

```diff
@@ -6,10 +6,10 @@ jobs:
   test:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v3
+    - uses: actions/checkout@v4
 
     - name: Setup Node
-      uses: actions/setup-node@v3
+      uses: actions/setup-node@v4
       with:
         node-version: 16.x
```

Comment:
shouldn't we update this node-version?

Reply:
We probably can, but don't have to, at least not in scope of this PR.
What the deprecation notice is about is a Node version used to run individual GH actions. Some types of actions (e.g. `actions/setup-node`) are built using Node. You can see the example of such config [here](https://github.com/actions/setup-node/blob/main/action.yml#L36). The `v4` of `actions/setup-node` uses `node20`, `v3` uses `node16`.
When we use
```
      uses: actions/setup-node@v4
      with:
        node-version: 16.x
```
the action itself will be build using `node20`, and once built, it will setup the Node version used to perform further steps of the workflow (running tests, installing) to version 16.
We may decide that we want to perform the tests/installation using a higher version of Node than v16, but this is a separate thing from making sure that our GH actions won't stop being supported due to that deprecation plan described in GH changelog.

@manumonti, I'm not familiar with the `merkle-distribiution`'s codebase, not sure if upgrading to Node 20 will not cause some issues. Do you want me to try?

Reply:
OK, should be good now...

Reply:
Oh, got it. These tests are not very important. They are testing a contract that was deployed a long time ago. I think we can leave it at that.

Thanks for the update and explanations!